test/unit/test_assembly.py

```diff
@@ -139,8 +139,7 @@ def test_lassa_protein(self):
         assembly.order_and_orient(
             os.path.join(inDir, 'contigs.lasv.fasta'),
             os.path.join(inDir, 'ref.lasv.fasta'),
-            outFasta,
-            aligner='promer')
+            outFasta)
```

Comment:
this is actually incorrect, but maybe we should just delete the test_lassa_protein unit test anyway since assembly.order_and_orient has not had this functionality for a while, and the unit test has just been skip-decorated this whole time.